web/src/components/pages/Integration/index.tsx

```diff
@@ -1,15 +1,18 @@
 import { AuthenticationRequiredPage } from "@reearth-cms/auth";
+import WorkspaceMenu from "@reearth-cms/components/molecules/Common/WorkspaceMenu";
 import Dashboard from "@reearth-cms/components/organisms/Dashboard";
 import Integration from "@reearth-cms/components/organisms/Settings/Integration";
 
-const IntegrationPage: React.FC = () => {
+const WorkspacePage: React.FC = () => {
```

Comment:
This is integrationPage, why are you renaming here and other integration pages to WorkspacePage?

Reply:
Fixed.
Typos, copied and pasted the name of the component by accident.